project_task_additional_attributes/models/project_task.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    account_manager = fields.Many2one("res.users")
```

Comment:
```suggestion
    account_manager_id = fields.Many2one("res.users")
```
This is in case this field should actually be added (instead of using `manager_id`).  Please check if this field is needed.